lib/Prefork.php

```diff
@@ -43,6 +48,11 @@ function __construct(array $args = array())
             ? $args['max_workers'] : $this->max_workers;
         $this->trap_signals = isset($args['trap_signals'])
             ? $args['trap_signals'] : $this->trap_signals;
+        $this->decide_action = isset($args['decide_action'])
+            ? $args['decide_action'] : null;
+        if ( isset($this->decide_action) && !is_callable($this->decide_action) ){
+            die("decide_action should be callable\n");
+        }
```

Comment:
What I wanted to say is:

* Define setter for `$decide_action`. e.g. `setDecideAction($function)`
* Validate `$function` above `is_callable()` in the setter.
* No need to accept `$args['decide_action']` in `__construct()` like above.
  * Or [like this](https://github.com/travail/php-Parallel-Prefork/pull/4/files#r98621874).
* Set empty lambda function to `$decide_action` in `__construct()` to ensure `$decide_action` is callable.

Simple usage is like this.

```php
$pp = new Prefork([
    ...,
]);
$pp->setDecideAction(function (...) use(...) { ... });
```